journal-of-neolithic-archaeology.csl

```diff
@@ -231,12 +231,14 @@
   <macro name="place-date">
     <choose>
       <if type="book thesis chapter paper-conference" match="any">
-        <choose>
-          <if match="none" variable="edition">
-            <text value=" "/>
-          </if>
-        </choose>
-        <group delimiter=" " prefix=" (" suffix=")">
+        <group delimiter="" prefix=" ">
+          <choose>
+            <if match="none" variable="edition">
+              <text macro="edition"/>
```

Comment:
This makes no sense -- if you don't have an edition variable, the edition macro is empty. What are you trying to do here?